src/main/java/uk/gov/hmcts/reform/prl/utils/CaseUtils.java

```diff
@@ -983,10 +983,10 @@ public static String getHearingId(WaMapper waMapper) {
     public static Optional<Long> getHearingId(WaMapper waMapper, Map<String, Object> caseDataUpdated) {
         Optional<Long> taskHearingId = ofNullable(waMapper)
             .filter(value ->
-                        new ObjectMapper().convertValue(
+                        Yes.equals(new ObjectMapper().convertValue(
                             caseDataUpdated.get(IS_INVOKED_FROM_TASK),
```

Comment:
I suppose you need a check for caseDataUpdated.get(IS_INVOKED_FROM_TASK) is not null before comparing the value

Reply:
I think no need to check null, I have added a test to check if caseDataUpdated.get(IS_INVOKED_FROM_TASK) is null then it doesn't throw exception while converting from null to YesNo type